ui/src/app/components/search-input.tsx

```diff
@@ -0,0 +1,225 @@
+import {TextInput} from 'app/components/inputs';
+import {TooltipTrigger} from 'app/components/popups';
+import colors, {colorWithWhiteness} from 'app/styles/colors';
+import {reactStyles} from 'app/utils';
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+
+const styles = reactStyles({
+  dropdownMenu: {
+    display: 'block',
+    maxHeight: '12rem',
+    minHeight: '30px',
+    visibility: 'visible',
+    overflowY: 'scroll',
+    width: '35%',
+    marginTop: '.25rem',
+    zIndex: 100,
+    border: '1px solid #979797',
+    borderRadius: '5px',
+    backgroundColor: colors.white,
+  },
+  open: {
+    position: 'absolute',
+    backgroundColor: colors.white,
+    border: '1px solid'
+  },
+  box: {
+    borderRadius: '5px',
+    paddingTop: '0.2rem',
+    paddingLeft: '0.2rem',
+    color: colors.primary,
+  },
+  boxHover: {
+    background: 'rgb(234, 243, 250)',
+    color: '#262262',
```

Comment:
@blrubenstein will be very happy if you use the shared colors class :)

Reply:
Fixed